addons/binding/org.openhab.binding.pidcontroller/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="pidcontroller"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>PIDController Binding</name>
+    <description>This is the binding for PIDController.</description>

```

Comment:
Any details about what devices specifically  are supported would be beneficial :)